src/manifest/manifest.ts

```diff
@@ -82,6 +84,11 @@ export class Manifest {
     this.createProjectConfigWatcher();
     const projectConfig = this.readAndParseProjectConfig();
 
+    const { pythonPath, onDidChangeExecutionDetails } = await getPythonPathFromExtention();
+    onDidChangeExecutionDetails(() => this.tryRefresh());

```

Comment:
There is a manifest file for each DBT project (monorepo setup), however there is only one venv per vscode window. Therefore this is not the right place. If the user has a lot of projects he will be prompted for each DBT project for DBT installation